### PR TITLE
miller 6.0.0

### PR DESCRIPTION
**Pull request by gromgit**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/miller.rb [hunk 1]
```diff
@@ -17,17 +17,13 @@ class Miller < Formula
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  depends_on "go" => :build
   depends_on "libtool" => :build
 
   uses_from_macos "flex" => :build
```
Comment (branchvincent) [inline Formula/miller.rb:23]: i don't think we need these anymore right?
```suggestion
  depends_on "go" => :build
```

Comment (johnkerl) [reply Formula/miller.rb:23]: @branchvincent correct ... thank you!

Comment (gromgit) [reply Formula/miller.rb]: Whoops, forgot about that. :grin:

**Review by branchv — APPROVED** [review]
thanks @gromgit, and @johnkerl for the quick help!

**Pull request closed without merging**